src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -454,6 +454,10 @@ def notifySlackBuildFailure(exception, channel) {
 }
 
 def versionHasIncremented(currVers, newVers) {
+  // first version is valid, i.e. currVers will be empty
```

Comment:
I'm not sure this comment is accurate as it might have been the version has been removed.

Reply:
How about something like `currVers will be empty on first merge to master`?

Reply:
Looks good. I've added another line to clarify.